esp/services/ws_dfu/ws_dfuXRefService.cpp

```diff
@@ -117,6 +117,10 @@ void CWsDfuXRefEx::init(IPropertyTree *cfg, const char *process, const char *ser
         throw MakeStringException(-1, "No Dali Connection Active. Please Specify a Dali to connect to in you configuration file");
     }
 
+#ifndef _CONTAINERIZED
+    initBareMetalRoxieTargets(roxieConnMap);
```

Comment:
It seems like there is another 
```
  void initBareMetalRoxieTargets(MapStringToMyClass<ISmartSocketFactory> &connMap, IPropertyTree *serviceTree, const char *daliAddress)
```
in ws_ecl_service.cpp but no defaults provided for serviceTree and daliAddress.
Is this ok ?

Reply:
OK yes you're right- I'd copied that function _from_ ws_ecl_service.cpp into TpWrapper for HPCC-31394 and missed the common-up. Should I common up in this ticket or in the parent (HPCC-32962) where I'll roll out the fix to the other services?

Reply:
I've added a note to HPCC-32962 to common-up that code.

Reply:
ok